README.md

```diff
@@ -141,11 +141,11 @@ DataFusion also includes a simple command-line interactive SQL utility. See the
 - [x] SQL Parser
 - [x] SQL Query Planner
 - [x] Query Optimizer
- - [x] Constant folding
- - [x] Join Reordering
- - [x] Limit Pushdown
- - [x] Projection push down
- - [x] Predicate push down
+- [x] Constant folding
+- [x] Join Reordering
+- [x] Limit Pushdown
+- [x] Projection push down
+- [x] Predicate push down
```

Comment:
I think these were meant to be sub bullets (they are features of the query optimizer) but we can fix that in a subsequent PR)